src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,44 @@
 
 class Animal {
   // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
+
+  static updateAliveList() {
+    Animal.alive = Animal.alive.filter(animal => animal.health > 0);
+  }
 }
 
+Animal.alive = [];
```

Comment:
I declare it as static object in class definition, linter is crying but in vsCode it's works

Reply:
My linter didn't let me do it :(